assets/sass/components/_scan-barcode-result.scss

```diff
@@ -0,0 +1,20 @@
+#scan-barcode-result-notification {
+
+  .govuk-notification-banner__header {
+    height: 25px;
+
+    h2.govuk-notification-banner__title {
+      @extend .govuk-visually-hidden
+    }
+  }
+
+  .success {
+    .govuk-notification-banner__content p {
+      // hook class to enable styling of success banner such as applying green tick
+    }
+  }
+
+  .notification {
+    // hook class to enable styling of notification banners
```

Comment:
This allows us to style the notification banner if we get asked to (such as making it orange rather than blue without changing the markup or the accessible-ness of it :) ). For the time being I've left this here as a hook point for as and when we are asked to do it